WordPress/Classes/ViewRelated/Notifications/Tools/KeyboardDismissHelper.swift

```diff
@@ -0,0 +1,191 @@
+import Foundation
+import UIKit
+
+
+/// Helper meant to aid in the InteractiveDismiss process.
+///
+/// Why this class exists:
+/// -   UIScrollView comes with a handy `UIScrollViewKeyboardDismissModeInteractive` keyboardDismissMode.
+/// -   During an interactive dismiss, no keyboard repositioning notification gets posted. Ever.
+/// -   Meant initially to be used along with ReplyTextView
+///
+/// Usage:
+/// -   Initialize it with a reference to the scrollView + bottom constraint + dismissableControl
+/// -   Forward the scrollView willBegin/didScroll events
+///
+@objc class KeyboardDismissHelper: NSObject
+{
+    /// Reference to the control to-be-dismissed
+    ///
+    var dismissableControl: UIView?
+
+    /// Reference to the bottom layout constraint
+    ///
+    var bottomLayoutConstraint: NSLayoutConstraint
+
+    /// Closure to be executed whenever the Keyboard will be Hidden
+    ///
+    var onWillHide: (Void -> ())?
```

Comment:
You're actually right!! `() -> Void` **IS** the right way. Thanks for pointing this out!